tests/integration/telemetry/stats/prometheus/http/stats.go

```diff
@@ -93,61 +99,93 @@ func TestSetup(ctx resource.Context) (err error) {
 	if err != nil {
 		return
 	}
-	bookinfoNsInst, err = namespace.New(ctx, namespace.Config{
-		Prefix: "istio-bookinfo",
+	appNsInst, err = namespace.New(ctx, namespace.Config{
+		Prefix: "echo",
 		Inject: true,
 	})
 	if err != nil {
 		return
 	}
-	if _, err = bookinfo.Deploy(ctx, bookinfo.Config{Namespace: bookinfoNsInst, Cfg: bookinfo.BookInfo}); err != nil {
-		return
+	if pilotInst, err = pilot.New(ctx, pilot.Config{
+		Galley: galInst,
+	}); err != nil {
+		return err
 	}
-	ingInst, err = ingress.New(ctx, ingress.Config{Istio: ist})
+
+	b, err := echoboot.NewBuilder(ctx)
 	if err != nil {
 		return
 	}
-	promInst, err = prometheus.New(ctx, prometheus.Config{})
+	err = b.
+		With(&client, echo.Config{
+			Service:   "client",
+			Namespace: appNsInst,
+			Ports:     nil,
+			Subsets:   []echo.SubsetConfig{{}},
+			Galley:    galInst,
+			Pilot:     pilotInst,
+		}).
+		With(&server, echo.Config{
+			Service:   "server",
+			Namespace: appNsInst,
+			Subsets:   []echo.SubsetConfig{{}},
+			Ports: []echo.Port{
+				{
+					Name:         "http",
+					Protocol:     protocol.HTTP,
+					InstancePort: 8090,
+				},
+			},
+			Galley: galInst,
+			Pilot:  pilotInst,
+		}).
+		Build()
 	if err != nil {
-		return
+		return err
 	}
-	bookingfoGatewayFile, err := bookinfo.NetworkingBookinfoGateway.LoadGatewayFileWithNamespace(bookinfoNsInst.Name())
+	promInst, err = prometheus.New(ctx, prometheus.Config{})
 	if err != nil {
 		return
 	}
-	err = galInst.ApplyConfig(
-		bookinfoNsInst,
-		bookingfoGatewayFile,
-	)
-	if err != nil {
-		return
+	// Application scraping will not work with permissive mode
+	if err = galInst.ApplyConfig(appNsInst, fmt.Sprintf(`
```

Comment:
I think there is cluster wide option to turn on strict mtls? Could you instead add those options to these two places:
https://github.com/istio/istio/blob/b3b0fd35cae8dfd0321a7141f5313cc300b2523d/tests/integration/telemetry/stats/prometheus/http/wasm/stats_wasm_filter_test.go#L52 https://github.com/istio/istio/blob/b3b0fd35cae8dfd0321a7141f5313cc300b2523d/tests/integration/telemetry/stats/prometheus/http/nullvm/stats_filter_test.go#L51

Reply:
This option has been removed, this is now the "correct" way to apply mTLS